api/client/node/update.go

```diff
@@ -47,7 +51,6 @@ func runUpdate(dockerCli *client.DockerCli, nodeID string, mergeNode func(node *
 		return err
 	}
 
-	fmt.Fprintf(dockerCli.Out(), "%s\n", nodeID)
```

Comment:
@dnephin because run update is also used in `node promote/demote/accept` see my example in the PR description, it prints 2 lines